### PR TITLE
Alter License Header

### PR DESCRIPTION
**Pull request by gregorydlogan**
This PR alters the license header so that it's not Javadocs anymore.  Since this will _doubtlessly_ end up with conflicts, here's the regex I used to make this:

`find ./ -type f -name *.java -exec sed -i '0,/\/\*\*/{s/\/\*\*/\/\*/}' "{}" \;`

### Your pull request should…

* [x] have a concise title
* [ ] ~[close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists~
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [ ] ~include migration scripts and documentation, if appropriate~
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: modules/admin-ui-frontend/app/styles/vendor/bourbon/css3/_background-image.scss
Comment (ziegenberg) [inline modules/admin-ui-frontend/app/styles/vendor/bourbon/css3/_background-image.scss:1]: wanna correct this one?

File: modules/asset-manager-api/src/main/java/org/opencastproject/assetmanager/api/query/package-info.java
Comment (ziegenberg) [inline modules/asset-manager-api/src/main/java/org/opencastproject/assetmanager/api/query/package-info.java:1]: this also needs correction.

File: modules/capture-admin-service-api/src/main/java/org/opencastproject/capture/admin/api/package-info.java
Comment (ziegenberg) [inline modules/capture-admin-service-api/src/main/java/org/opencastproject/capture/admin/api/package-info.java:1]: This needs an update.

File: modules/common/src/main/java/org/opencastproject/serviceregistry/api/package-info.java
Comment (ziegenberg) [inline modules/common/src/main/java/org/opencastproject/serviceregistry/api/package-info.java:1]: This needs an update.

File: modules/common/src/main/java/org/opencastproject/util/package-info.java
Comment (ziegenberg) [inline modules/common/src/main/java/org/opencastproject/util/package-info.java:1]: This needs an update.

**Review by ziegenberg — CHANGES_REQUESTED** [review]

**Comment by ziegenberg** [comment]
I'm only half way through the changes ... Hopefully will have time to check the others too

File: modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/query/package-info.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/**
```
Comment (ziegenberg) [inline modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/query/package-info.java:1]: This has the wrong license header and the documentation.

File: modules/composer-service-api/src/main/java/org/opencastproject/composer/layout/package-info.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/**
```
Comment (ziegenberg) [inline modules/composer-service-api/src/main/java/org/opencastproject/composer/layout/package-info.java:1]: This was not a license header.

File: modules/distribution-service-api/src/main/java/org/opencastproject/distribution/api/package-info.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/**
```
Comment (ziegenberg) [inline modules/distribution-service-api/src/main/java/org/opencastproject/distribution/api/package-info.java:1]: This was not a license header.

File: modules/distribution-service-download/src/main/java/org/opencastproject/distribution/download/package-info.java [hunk 4]
```diff
@@ -1,4 +1,4 @@
-/**
```
Comment (ziegenberg) [inline modules/distribution-service-download/src/main/java/org/opencastproject/distribution/download/package-info.java:1]: This was not a license header.

File: modules/ingest-service-impl/src/test/java/org/apache/commons/fileupload/MockHttpServletRequest.java [hunk 5]
```diff
@@ -1,5 +1,5 @@
 //CHECKSTYLE:OFF
-/**
```
Comment (ziegenberg) [inline modules/ingest-service-impl/src/test/java/org/apache/commons/fileupload/MockHttpServletRequest.java:2]: This should probably not be modified.

File: modules/search-service-api/src/main/java/org/opencastproject/search/api/package-info.java [hunk 6]
```diff
@@ -1,4 +1,4 @@
-/**
```
Comment (ziegenberg) [inline modules/search-service-api/src/main/java/org/opencastproject/search/api/package-info.java:1]: This was not a license header.

File: modules/search-service-impl/src/main/java/org/opencastproject/search/impl/package-info.java [hunk 7]
```diff
@@ -1,4 +1,4 @@
-/**
```
Comment (ziegenberg) [inline modules/search-service-impl/src/main/java/org/opencastproject/search/impl/package-info.java:1]: This was not a license header.

File: modules/search-service-impl/src/main/java/org/opencastproject/search/impl/persistence/package-info.java [hunk 8]
```diff
@@ -1,4 +1,4 @@
-/**
```
Comment (ziegenberg) [inline modules/search-service-impl/src/main/java/org/opencastproject/search/impl/persistence/package-info.java:1]: This was not a license header.

File: modules/series-service-api/src/main/java/org/opencastproject/series/api/package-info.java [hunk 9]
```diff
@@ -1,4 +1,4 @@
-/**
```
Comment (ziegenberg) [inline modules/series-service-api/src/main/java/org/opencastproject/series/api/package-info.java:1]: This was not a license header.

Also, I like the lonely "TODO" in here ;)

File: modules/series-service-impl/src/main/java/org/opencastproject/series/endpoint/package-info.java [hunk 10]
```diff
@@ -1,4 +1,4 @@
-/**
```
Comment (ziegenberg) [inline modules/series-service-impl/src/main/java/org/opencastproject/series/endpoint/package-info.java:1]: This was not a license header.

File: modules/series-service-impl/src/main/java/org/opencastproject/series/impl/package-info.java [hunk 11]
```diff
@@ -1,4 +1,4 @@
-/**
```
Comment (ziegenberg) [inline modules/series-service-impl/src/main/java/org/opencastproject/series/impl/package-info.java:1]: This was not a license header.

File: modules/series-service-impl/src/main/java/org/opencastproject/series/impl/persistence/package-info.java [hunk 12]
```diff
@@ -1,4 +1,4 @@
-/**
```
Comment (ziegenberg) [inline modules/series-service-impl/src/main/java/org/opencastproject/series/impl/persistence/package-info.java:1]: This was not a license header.

**Comment by ziegenberg** [comment]
Before littering the comments even more, you might wanna look into the following files:

```
modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/query/package-info.java
modules/composer-service-api/src/main/java/org/opencastproject/composer/layout/package-info.java
modules/distribution-service-api/src/main/java/org/opencastproject/distribution/api/package-info.java
modules/distribution-service-download/src/main/java/org/opencastproject/distribution/download/package-info.java
modules/ingest-service-impl/src/test/java/org/apache/commons/fileupload/MockHttpServletRequest.java
modules/search-service-api/src/main/java/org/opencastproject/search/api/package-info.java
modules/search-service-impl/src/main/java/org/opencastproject/search/impl/package-info.java
modules/search-service-impl/src/main/java/org/opencastproject/search/impl/persistence/package-info.java
modules/series-service-api/src/main/java/org/opencastproject/series/api/package-info.java
modules/series-service-impl/src/main/java/org/opencastproject/series/endpoint/package-info.java
modules/series-service-impl/src/main/java/org/opencastproject/series/impl/package-info.java
modules/series-service-impl/src/main/java/org/opencastproject/series/impl/persistence/package-info.java
modules/static-file-service-api/src/test/java/org/apache/commons/fileupload/MockHttpServletRequest.java
modules/statistics-provider-influx/src/main/java/org/opencastproject/statistics/provider/influx/package-info.java
modules/statistics-service-api/src/main/java/org/opencastproject/statistics/api/package-info.java
modules/statistics-service-impl/src/main/java/org/opencastproject/statistics/endpoint/package-info.java
modules/statistics-service-impl/src/main/java/org/opencastproject/statistics/impl/package-info.java
modules/usertracking-api/src/main/java/org/opencastproject/usertracking/api/package-info.java
modules/usertracking-impl/src/main/java/org/opencastproject/usertracking/endpoint/package-info.java
modules/usertracking-impl/src/main/java/org/opencastproject/usertracking/impl/package-info.java
modules/workflow-service-api/src/main/java/org/opencastproject/workflow/api/package-info.java
modules/workflow-service-impl/src/main/java/org/opencastproject/workflow/endpoint/package-info.java
modules/workspace-api/src/main/java/org/opencastproject/workspace/api/package-info.java
```

**Comment by gregorydlogan** [comment]
Current comments addressed.  There are a bunch more package-info files without headers which will be addressed shortly.

File: modules/series-service-impl/src/main/java/org/opencastproject/series/impl/package-info.java [hunk 13]
```diff
@@ -1,4 +1,25 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+
+ /*
```
Comment (ziegenberg) [inline modules/series-service-impl/src/main/java/org/opencastproject/series/impl/package-info.java]: This was a JavaDoc before. Is it on purpose to demote it to a regular comment? See also the other package-info classes.

Comment (gregorydlogan) [reply modules/series-service-impl/src/main/java/org/opencastproject/series/impl/package-info.java]: Yep, the goal is to convert all of these into regular comments.  If there are some missing, or some which are still javadocs please point them out

Comment (ziegenberg) [reply modules/series-service-impl/src/main/java/org/opencastproject/series/impl/package-info.java]: I'm not talking about the license header but the JavaDoc directly above the `package Xxxxxx;` line. Just to be sure, we're talking about the same thing...


> Yep, the goal is to convert all of these into regular comments. If there are some missing, or some which are still javadocs please point them out

In this case, the following files also need to be changed:

```
modules/asset-manager-api/src/main/java/org/opencastproject/assetmanager/api/query/package-info.java
modules/asset-manager-api/src/main/java/org/opencastproject/assetmanager/api/package-info.java
modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/persistence/package-info.java
modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/query/package-info.java
modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/package-info.java
modules/capture-admin-service-api/src/main/java/org/opencastproject/capture/admin/api/package-info.java:
modules/capture-admin-service-impl/src/main/java/org/opencastproject/capture/admin/endpoint/package-info.java
modules/capture-admin-service-impl/src/main/java/org/opencastproject/capture/admin/impl/package-info.java
modules/common/src/main/java/org/opencastproject/serviceregistry/api/package-info.java
modules/common/src/main/java/org/opencastproject/util/package-info.java
```

**Comment by gregorydlogan** [comment]
@ziegenberg ok, think I've got it all this time.  Went through the diff manually locally (so much faster than GH) and the remaining changes *should* be just the header bits.

**Review by ziegenberg — APPROVED** [review]
This looks good now. Ready to merge :)

**Pull request merged**